crates/ra_mbe/src/syntax_bridge.rs

```diff
@@ -63,56 +85,92 @@ fn fragment_to_syntax_node(
     };
     let buffer = TokenBuffer::new(&tokens);
     let mut token_source = SubtreeTokenSource::new(&buffer);
-    let mut tree_sink = TtTreeSink::new(buffer.begin());
+    let mut range_map = ExpandedRangeMap::default();
+    let mut tree_sink = TtTreeSink::new(buffer.begin(), &mut range_map);
```

Comment:
Maybe return range_map from .finish() instead?

Reply:
Yes, I agree. Added a `finish` method to return it.